docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -28,6 +28,12 @@ const appList = [
   //   link: '',
   // },
   {
+    title: 'SlimChess - Play Instant Chess Online Games on the Go',
```

Comment:
We add the examples by chronological order. Could you add it at the end? We will reorginize the examples at some point once we have enough high quality one.